docker-compose.yml

```diff
@@ -29,6 +29,7 @@ services:
       http.cors.allow-headers: X-Requested-With,X-Auth-Token,Content-Type,Content-Length,Authorization,test,access-control-allow-headers,x-search-query,X-Search-Id,X-Search-Filters,X-ubl-query-id,X-ubl-store,X-ubl-user-id,X-ubl-session-id,x-search-client,Access-Control-Expose-Headers
       # indexes faster if we don't log each row (i.e. logger.level: debug/info)
       logger.level: info
+      OPENSEARCH_INITIAL_ADMIN_PASSWORD: Mysupersecretpassword_123
```

Comment:
This is now required by OS 2.12. [Setting up a custom admin password](https://opensearch.org/docs/latest/security/configuration/demo-configuration/#setting-up-a-custom-admin-password)